src/libraries/Common/tests/System/Net/EnterpriseTests/setup/kdc/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.0.100-bionic
```

Comment:
Is dotnet sdk required for this container? Looks like it could work with a vanilla ubuntu image.

Reply:
>Is dotnet sdk required for this container?

It isn't required. It was when I was first developing this solution and I was using the prior technique of having the tests build similar to the stress app (i.e. with public 'dotnet' tools). But I moved away from that approach.

I will  adjust this container to be based on regular ubuntu image in a follow-up PR as you suggest.